deluge/core/alertmanager.py

```diff
@@ -72,11 +102,17 @@ def update(self):
         self.delayed_calls = [dc for dc in self.delayed_calls if dc.active()]
         self.handle_alerts()
```

Comment:
You'll want to remove this line.  Currently the update() method is called every ~0.3 seconds (see line 84) by the Component system.

Reply:
I'm a little paranoid about this, since I'm not sure I've caught all the cases of the component lifecycle.